### PR TITLE
Update renault documentation - Add button section

### PR DESCRIPTION
**Pull request by rodriguestiago0**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  [PR](https://github.com/home-assistant/core/pull/88003)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -119,7 +119,6 @@ GEM
     tzinfo-data (1.2022.7)
       tzinfo (>= 1.0.0)
     unicode-display_width (2.4.2)
```
Comment (epenet) [inline Gemfile.lock]:
```suggestion
    unicode-display_width (2.4.2)
    webrick (1.8.1)
```

File: Gemfile.lock [hunk 2]
```diff
@@ -140,6 +139,7 @@ DEPENDENCIES
   stringex (= 2.8.5)
   tzinfo (~> 2.0)
   tzinfo-data
+  wdm (~> 0.1.0)
```
Comment (epenet) [inline Gemfile.lock]:
```suggestion
```

File: source/_integrations/renault.markdown [hunk 3]
```diff
@@ -37,6 +37,24 @@ This integration provides the following platforms:
 
 All vehicles linked to the account should then get added as devices, with sensors added as linked entity.
 
+## Buttons
+
+### Button `button.renault_start_charge`
+
+Start charging the vehicle.
```
Comment (epenet) [inline source/_integrations/renault.markdown]: maybe it is sufficient to insert a line between 31/32

Comment (rodriguestiago0) [reply source/_integrations/renault.markdown]: done

Comment (epenet) [reply source/_integrations/renault.markdown]: not done

Comment (rodriguestiago0) [reply source/_integrations/renault.markdown]: got an error when trying to push. sorry

File: source/_integrations/renault.markdown [hunk 4]
```diff
@@ -30,6 +30,7 @@ This integration provides the following platforms:
 
 - Binary sensors - such as plug and charge status.
 - Device tracker - to track location of your car.
+- Buttons - to start/stop charge. (Button action can be unavailable for your vehicle.)
```
Comment (epenet) [inline source/_integrations/renault.markdown]:
```suggestion
- Buttons - to start air conditioning or start/stop the charge. Please note that although available these actions do not work on all vehicles.
```

**Review by epenet — APPROVED** [review]
LGTM 👍

**Pull request merged**